zad3/app/controllers/bestiary_controller.rb

```diff
@@ -2,13 +2,25 @@
 
 class BestiaryController < ApplicationController
   def index
-
+    require monsters
+    create table :Monsters 
```

Comment:
oookej, zgaduję że szukałeś czegoś w stylu `how to create table in rails` i natrafiłeś na metodę `create_table`. Służy ona jednak do czegoś zupełnie innego niż tworzenie HTMLa - używa się jej w migracjach do tworzenia tabel w bazie danych 😉